.eslintrc.js

```diff
@@ -239,6 +239,10 @@ module.exports = {
         selector: "CallExpression[callee.name='isNaN']",
         message: 'Use Number.isNaN() instead of the global isNaN() function.',
       },
+      {
```

Comment:
Can you add a TODO here with the link to the eslint bug, to remove this when `no-restricted-properties` is fixed?